### DISCUSSION TITLE
How about GO's wails and Rust's tauri ? There are JavaScript in all of them.

### DISCUSSION THREAD

**Discussion by CHENKAI00** [ShellWen/v8_killer#26] [Q&A]
How about GO's wails and Rust's tauri ? There are JavaScript in all of them.

**Comment by ShellWen** [comment]
I think it's okay, but I haven't tested it

**Comment by CHENKAI00** [comment]
I am learning Rust. After I will test.

**Comment by ghostflyby** [comment]
Both frameworks utilize webkit on linux and macOS, whose javascript engine is JavaScriptCore instead of v8.
